server/poll/transform.go

```diff
@@ -208,3 +224,41 @@ func (p *Poll) ToEndPollPost(bundle *utils.Bundle, authorName string, convert ID
 
 	return post, nil
 }
+
+// ToCard return the poll for rhs card
+func (p *Poll) ToCard(bundle *utils.Bundle, convert IDToNameConverter) string {
+	localizer := bundle.GetServerLocalizer()
+	s := fmt.Sprintf("# %s\n", p.Question)
+
+	creatorName, _ := convert(p.Creator)
+	s += fmt.Sprintf(localizer.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: rhsCardPollCreatedBy})+" %s\n", creatorName)
```

Comment:
Please use `bundle.LocalizeWithConfig` instead of `MustLocalize`, because `MustLocalize` causes panic if translation is missing.

refs: https://github.com/matterpoll/matterpoll/pull/404/files#diff-e4bd7a74772fffef524851fbf869db31032434192f155ad853eb40246915f760R61

Reply:
Done